lua/neogen/configurations/c.lua

```diff
@@ -104,6 +104,20 @@ return {
         },
     },
 
+	locator = function(node_info, nodes_to_match)
+		local result = neogen.default_locator(node_info, nodes_to_match)
+		if not result then
+			return nil
+		end
+		-- if the function happens to be a function template we want to place
+		-- the annotation before the template statement and extract the
+		-- template parameters names as well
+		if node_info.current:parent():type() == "template_declaration" then
```

Comment:
I think this part should be extracted in something like this:
```lua
if node_info.current:parent() == nil then
    return
end
```

So that it does not fail when there is no parent

Reply:
Changed, hopefully the way you meant it.

Reply:
Yes exactly !